src/main/java/iguanaman/hungeroverhaul/config/IguanaConfig.java

```diff
@@ -241,7 +244,7 @@ public static <T> ConfigOption<T> putOptionInList(ConfigCategory category, Strin
     public static int respawnHungerDifficultyModifier;
     public static final ConfigOption<Boolean> disableHealingHungerDrainOption = addOption(CATEGORY_HUNGER, "disableHealingHungerDrain", true, false, "Disable the hunger drain when healing that was introduced in vanilla 1.6.2");
     public static boolean disableHealingHungerDrain;
-    public static final ConfigOption<Float> hungerLossRatePercentageOption = addOption(CATEGORY_HUNGER, "hungerLossRatePercentage", 4F/3F * 100F, 0F, null, 100F, "Speed up or slow down the rate that hunger drops (set to 0 to disable hunger loss)");
+    public static final ConfigOption<Float> hungerLossRatePercentageOption = addOption(CATEGORY_HUNGER, "hungerLossRatePercentage", 100F, 0F, null, 100F, "Speed up or slow down the rate that hunger drops (set to 0 to disable hunger loss)");
```

Comment:
Why change the default here from 133% to 100%?

Reply:
To keep a similar level of difficulty in HARD while making the difficulty curve linear.

With this chart it should be easy to comprehend why the base config needs to be changed:
![image](https://user-images.githubusercontent.com/47131096/216989596-1d7e2579-0a6a-482e-be76-8ced01669711.png)

### Why do so?
There are players who play the pack to enjoy themselves (NORMAL) and those who want to experience the c*** and b**** torture to challenge themselves to gain their lost father's approval (HARD).

Where before both had to deal with the same difficulty level this commit changes that.

This allows us to better tailor each experience as the food lasts longer for everyone else while a little less for those in HARD mode (3.33% shorter)